handsontable/test/e2e/core/getRowHeight.spec.js

```diff
@@ -26,4 +26,37 @@ describe('Core.getRowHeight', () => {
     expect(modifyRowHeight.calls.count()).toBe(1);
     expect(modifyRowHeight).toHaveBeenCalledWith(50, 1);
   });
+
+  it('should return `rowHeights` if value is greater than `defaultHeight`', () => {
```

Comment:
WDYT about adding a more precise description? (mainly changing "defaultHeight" to "minimal theme row height")
```suggestion
  it('should return the same value as the `rowHeights` if the value is greater than minimum theme row height', () => {
```